src/main/java/com/vaadin/demo/component/datetimepicker/DateTimePickerInternationalization.java

```diff
@@ -0,0 +1,34 @@
+package com.vaadin.demo.component.datetimepicker;
+
+import java.util.List;
+
+import com.vaadin.demo.DemoExporter;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.datepicker.DatePicker;
+import com.vaadin.flow.component.datetimepicker.DateTimePicker;
+import com.vaadin.flow.router.Route;
+
+@Route("date-time-picker-internationalization")
+public class DateTimePickerInternationalization extends Div {
+
+    public DateTimePickerInternationalization() {
+        DateTimePicker dateTimePicker = new DateTimePicker("Sitzungsdatum");
+
+        // tag::snippet[]
+        DatePicker.DatePickerI18n customI18n = new DatePicker.DatePickerI18n();
```

Comment:
```suggestion
        DatePicker.DatePickerI18n germanI18n = new DatePicker.DatePickerI18n();
```
Makes it less confusing what kind of language is actually being used.